README.md

```diff
@@ -215,28 +212,49 @@ import 'package:flutter_credit_card/flutter_credit_card.dart';
       },
       autovalidateMode: AutovalidateMode.always,
       disableCardNumberAutoFillHints: false,
-      cardNumberDecoration: const InputDecoration(
-        border: OutlineInputBorder(),
-        labelText: 'Number',
-        hintText: 'XXXX XXXX XXXX XXXX',
-      ),
-      expiryDateDecoration: const InputDecoration(
-        border: OutlineInputBorder(),
-        labelText: 'Expired Date',
-        hintText: 'XX/XX',
-      ),
-      cvvCodeDecoration: const InputDecoration(
-        border: OutlineInputBorder(),
-        labelText: 'CVV',
-        hintText: 'XXX',
-      ),
-      cardHolderDecoration: const InputDecoration(
-        border: OutlineInputBorder(),
-        labelText: 'Card Holder',
+      inputConfiguration: const InputConfiguration(
+        cardNumberDecoration: const InputDecoration(
+          border: OutlineInputBorder(),
+          labelText: 'Number',
+          hintText: 'XXXX XXXX XXXX XXXX',
+        ),
+        expiryDateDecoration: const InputDecoration(
+          border: OutlineInputBorder(),
+          labelText: 'Expired Date',
+          hintText: 'XX/XX',
+        ),
+        cvvCodeDecoration: const InputDecoration(
+          border: OutlineInputBorder(),
+          labelText: 'CVV',
+          hintText: 'XXX',
+        ),
+        cardHolderDecoration: const InputDecoration(
+          border: OutlineInputBorder(),
+          labelText: 'Card Holder',
+        ),
+        cardNumberTextStyle: TextStyle(
+          fontSize: 10,
+          color: Colors.black,
+        ),
+        cardHolderTextStyle: TextStyle(
+          fontSize: 10,
+          color: Colors.black,
+        ),
+        expiryDateTextStyle: TextStyle(
+          fontSize: 10,
+          color: Colors.black,
+        ),
+        cvvCodeTextStyle: TextStyle(
+          fontSize: 10,
+          color: Colors.black,
+        ),
       ),
     ),
 ```
 
+## Breaking Changes in 4.x.x
+- `themeColor`, `textColor`, and `cursorColor` have been removed from `CreditCardForm` as it can't detect the application theme. Please check out the example app to learn how to apply those using `Theme`.
```

Comment:
Please mention removal of input decoration fields and how to use in the newer version.